src/drivetrain/drivetrain/drivetrain_node.py

```diff
@@ -11,22 +11,28 @@
 
 # Import ROS 2 formatted message types
 from geometry_msgs.msg import Twist
+from std_msgs.msg import Float64
 
 # Import custom ROS 2 interfaces
 from rovr_interfaces.srv import Stop, Drive, MotorCommandSet
 from rovr_interfaces.msg import AbsoluteEncoders
 
+simulation = True
```

Comment:
we don't need this variable here anymore right?

Reply:
yeah i have a commit removing it